proteus/BC.py

```diff
@@ -93,10 +93,7 @@ def setParallelFlag0(self):
         This function should be used only on BC flag 0 (index 0 in the list
         domain.bc)
         """
-        self.vof_advective = constantBC(0.)
-        self.u_diffusive = constantBC(0.)
-        self.v_diffusive = constantBC(0.)
-        self.w_diffusive = constantBC(0.)
+        self.reset()
```

Comment:
We need to set these vof advective and u diffusive to 0 for parallel runs, otherwise the log gets spammed with errors and sets these values to 0 by itself